hypertrace-graphql-label-application-rules-schema/src/main/java/org/hypertrace/graphql/label/application/rules/dao/LabelApplicationRuleRequestConverter.java

```diff
@@ -47,11 +48,16 @@ public DeleteLabelApplicationRuleRequest convertDeleteRequest(
 
   private LabelApplicationRuleData convertLabelApplicationRuleData(
       org.hypertrace.graphql.label.application.rules.schema.shared.LabelApplicationRuleData data) {
-    return LabelApplicationRuleData.newBuilder()
-        .setName(data.name())
-        .setMatchingCondition(convertConditionList(data.conditionList()))
-        .setLabelAction(convertLabelAction(data.action()))
-        .build();
+    LabelApplicationRuleData.Builder convertedDataBuilder =
+        LabelApplicationRuleData.newBuilder()
+            .setName(data.name())
+            .setMatchingCondition(convertConditionList(data.conditionList()))
+            .setLabelAction(convertLabelAction(data.action()))
+            .setEnabled(data.enabled());
+    if (Objects.nonNull(data.description())) {
+      convertedDataBuilder.setDescription(Objects.requireNonNull(data.description()));
```

Comment:
You have already done a null check, so here you can just use `data.description()`

Reply:
It indicates the programmer's intent that the field should be non-null. Easier to read and understand. 
Moreover, the IDE warns if I don't enclose it in requireNonNull.

Reply:
I think @saxenakshitiz 's point is that the if check on line 57 already does that, we're doing the same thing twice. I wouldn't expect the IDE to flag it after that, but it highlights why I prefer the suggestion from https://github.com/hypertrace/hypertrace-graphql/pull/116#discussion_r750308298 - it doesn't require one line of code to depend on the logic of another line - the compiler enforces it.

Reply:
Done